repo/packages/B/beta-tensorflow/2/config.json

```diff
@@ -0,0 +1,410 @@
+{
+  "type": "object",
+  "properties": {
+    "service": {
+      "type": "object",
+      "description": "DC/OS service configuration properties",
+      "properties": {
+        "name": {
```

Comment:
@elezar It seems we removed the "title" field from all the properties like  "title": "Service name" which has the description - "description": "The name of the TensorFlow service instance." Is there a reason for that?

Reply:
I will double check tomorrow why this happened. I think we in general don't use the title field for the config schemas.

Reply:
I have confirmed that I removed this mesosphere/dcos-tensorflow#43 while aligning the TF config.json with that of the other frameworks where we don't use the "title".